SECURITY.md

```diff
@@ -76,4 +105,31 @@ Extensions configuration setup is tied to OpenSearch settings, extensions config
+
+Additional background avaliable from [Security#1895](https://github.com/opensearch-project/security/issues/1895)
+
+### User identity [OpenSearch#3846](https://github.com/opensearch-project/OpenSearch/issues/3846) :negative_squared_cross_mark:
+Replace [commons.authuser.User](https://github.com/opensearch-project/common-utils/blob/main/src/main/java/org/opensearch/commons/authuser/User.java) the common identity object used by plugins.  The new object should be obtainable by the extension through other means than [InjectSecurity](https://github.com/opensearch-project/common-utils/blob/main/src/main/java/org/opensearch/commons/InjectSecurity.java#L69) depends on the thread context.
+
+#### Aquiring User objects [sdk#37](https://github.com/opensearch-project/opensearch-sdk/issues/37)
+When OpenSearch sents a request to an extension the identiy should be avaliable, more discussion see [Handling identity in extensions](https://github.com/opensearch-project/opensearch-sdk/issues/14).
+
+#### Resource user/role checks [sdk#40](https://github.com/opensearch-project/opensearch-sdk/issues/40)
+anomaly Detection has detectors that analyzer data and store its results so it can be inspected or alerted on, [more details](https://opensearch.org/docs/latest/monitoring-plugins/ad/index/). OpenSearch should be responsible for inspecting the user, roles, resources to ensure standard practices are used.  A permissions check API should be designed and implemented to offload this work from extensions creators.
+
+### Delegate Authority  [OpenSearch#3850](https://github.com/opensearch-project/OpenSearch/issues/3850) :negative_squared_cross_mark:
+anomaly Detection runs background jobs to scan for anamolies and alerts that trigger if conditions are detected.  Background tasks should be tied to an idenity and a delegated identity so permissions can be verified.  The underlying systems depends on the [Job Scheduler](https://github.com/opensearch-project/job-scheduler/blob/main/src/main/java/org/opensearch/jobscheduler/scheduler/JobScheduler.java) plugin to execute these requests. 
+
+#### Extension identity [sdk#41](https://github.com/opensearch-project/opensearch-sdk/issues/41)
+There should be different levels of permissions granularity interactive allowing for disgushing a user actions or user action through an extension.  Extensions should have an identity and there should be a way that the identity of action is layered with all the parties that have triggered it. 
+
+#### Delayed action API [sdk#42](https://github.com/opensearch-project/opensearch-sdk/issues/42)
+When actions are triggered without an interactive user session OpenSearch will need to permit the action to occur or not.  Create an API for these background tasks to get an identity associated with the session.
```

Comment:
Probably want to mention `Principal` identities here.

Reply:
What do you think about moving this detail into the associated issue?  Its a great requirement that should be part of use cases and acceptance criteria.